packages/flutter_tools/lib/src/android/android_device.dart

```diff
@@ -214,7 +214,7 @@ class AndroidDevice extends Device {
 
   AdbLogReader _logReader;
   AdbLogReader _pastLogReader;
-  _AndroidDevicePortForwarder _portForwarder;
+  AndroidDevicePortForwarder _portForwarder;
 
   List<String> adbCommandForDevice(List<String> args) {
```

Comment:
I think there's only one more usage of this left, if you want to kill it.

Reply:
Theres a few more...